socorro/unittest/processor/test_processor_2015.py

```diff
@@ -63,64 +39,30 @@ def test_rule_sets_from_string_1(self):
         assert rc.ruleset01.rule_system_class.default == expected
         assert 'apply_all_rules' == rc.ruleset01.action.default
         expected = (
-            'socorro.processor.support_classifiers.BitguardClassifier, '
-            'socorro.processor.support_classifiers.OutOfDateClassifier'
+            'socorro.processor.general_transform_rules.CPUInfoRule, '
+            'socorro.processor.general_transform_rules.OSInfoRule '
```

Comment:
These classifiers are pulled in for the test but not applied. Near as I can tell they are used to verify that the config loads the classes.

Reply:
I concur.